ui/src/message_popup/renderers/minimal_timed_view.jsx

```diff
@@ -0,0 +1,88 @@
+/* @flow weak */
+import React from 'react';
+import RaisedButton from 'material-ui/RaisedButton';
+
+var sceneTimer;
+
+// This response supports responding with text.
+export default React.createClass({
+  displayName: 'MinimalTimedView',
+
+  propTypes: {
+    onLogMessage: React.PropTypes.func.isRequired,
+    onResponseSubmitted: React.PropTypes.func.isRequired,
+    recordText: React.PropTypes.string,
+  },
+
+  getDefaultProps() {
+    return {
+      recordText: 'Respond',
+      autoFocus: true,
+      textHeight: 48
+    };
+  },
+
+
+  getInitialState() {
+    sceneTimer = setTimeout(this.onTimer, 25000);
+    return {
+      responseText: 'type here!'
```

Comment:
This piece of state isn't used in this component, this is left over from copy-pasting.  You can delete this (and with the other comment above) you can remove `getInitialState` altogether.